packages/angular_devkit/build_angular/src/webpack/plugins/karma.ts

```diff
@@ -334,6 +334,22 @@ function fallbackMiddleware() {
   };
 }
 
+/**
+ * Returns a function that returns true if the plugin identifier matches the
+ * `moduleId` or `pluginName`. A plugin identifier can be either a string or
+ * an object according to https://karma-runner.github.io/5.2/config/plugins.html
+ * @param moduleId name of the node module (e.g. karma-coverage)
+ * @param pluginName name of the karma plugin (e.g. reporter:coverage)
+ */
+function isPlugin(moduleId: string, pluginName: string) {
+  return function(plugin: string|{}) {
```

Comment:
Nit: I think adding explicit return type is a better practice.

```suggestion
  return (plugin: string|{}): boolean  => {
```